rest_api/tests/unit/run_unit_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+python3 /project/sawtooth-core/rest_api/tests/unit/mock_validator.py &
```

Comment:
In my mind, there is no reason to not run multiple processes in one container, especially since they are meant to be unit tests. I think as long as the Rest API container is being tested as a docker container in the integration tests (which intkey and xo smoke do) having unit tests run in a single container is justified. We are already doing this in `signing/tests/unit_signing.yaml` and `cli/tests/unit_cli.yaml`.